ci/test/05_before_script.sh

```diff
@@ -16,6 +16,12 @@ fi
 DOCKER_EXEC mkdir -p ${DEPENDS_DIR}/SDKs ${DEPENDS_DIR}/sdk-sources
 
 DOCKER_EXEC test/functional/install-deps.sh
+retVal=$?
+if [ $retVal -eq 1 ]; then
+    echo "Error installing dependencies"
+    exit $retVal
+fi
+exit $retVal
```

Comment:
I suspect that this `exit` currently prevents further execution of the CI and should be removed.

Reply:
Now that you mention it, I strongly suspect that this extra `exit` is suspect.